.eslintignore

```diff
@@ -2,4 +2,5 @@ app/scripts/lib/extension-instance.js
 test/integration/bundle.js
 test/integration/jquery-3.1.0.min.js
 test/integration/helpers.js
-test/integration/lib/first-time.js
\ No newline at end of file
+test/integration/lib/first-time.js
+ui/lib/blockies.js
```

Comment:
Is it necessary to ignore your new file in the linting?

Reply:
Excellent question. This is the lib file from the MyEtherWallet project. I originally did not leave this file out of the eslint checks but it was blowing up Lint with a ton of styling preference "errors" (dangling commas etc). Apparently the MEW folks didn't follow the same style guide ;) I was faced with a dilemma: edit a library containing trusted code to fix some style issues, or omit the file from lint checking. I chose to leave it out of the checks.